framework/kms-keyring.md

```diff
@@ -235,7 +237,7 @@ If the call succeeds, OnEncrypt MUST do the following with the response from [KM
 - append a new [record](#structures.md#record) to the [keyring trace](#structures.md#keyring-trace)
   in the input [encryption materials](#structures.md#encryption-materials), constructed as follows:
   - The string field KeyNamespace contains "aws-kms".
-  - The string field KeyName contains [generator](#generator).
+  - The string field KeyName contains the CMK ARN used for [KMS Encrypt](#kms-encrypt).
```

Comment:
Both the C and JS ESDKs do not do this.  They both use the value returned from KMS.